uint8 QR = 0
uint8 NUMBER = 1
uint8 BOTH = 2
uint8 what
bool to